sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/_encode.py

```diff
@@ -295,9 +289,8 @@ def encode_long(output, value, with_constructor=True, use_smallest=True):
 
 
 def encode_float(
-    output, value, with_constructor=True, **kwargs  # pylint: disable=unused-argument
-):
-    # type: (bytearray, float, bool, Any) -> None
+    output: bytearray, value: float, with_constructor: bool = True, **kwargs: Any  # pylint: disable=unused-argument
```

Comment:
is unused args still needed here?

Reply:
its because of the unused `kwargs`

Reply:
You could probably replace `kwargs` with `_` throughout if we wanted to clean up all those pylint disables. Alternatively you could just move the statement to the top of the file so we only need it once. I think that would be safe enough.